HIRS_Utils/src/test/java/hirs/data/persist/TPMBaselineTest.java

```diff
@@ -623,6 +623,15 @@ public final void testSetTPMInfo() {
         Assert.assertEquals(baseline.getTPMInfo(), tpmInfo);
     }
 
+    /**
+     * Test whether a baseline has data or not.
+     */
+    @Test
+    public final void testIsEmpty() {
```

Comment:
Yeah so to complement this test, can we have this be `testIsEmptyFalse`, and the a have `testIsEmptyTrue` where you just create the a TPMBaseline with its default empty constructor, and test the Assertion to verify that `IsEmpty` is true.